src/Microsoft.PowerShell.Commands.Utility/commands/utility/XmlCommands.cs

```diff
@@ -120,6 +120,8 @@ public SwitchParameter NoClobber
 
```

Comment:
@IISResetMe, your last commit had 4 failures in `PowerShell-CI-macos`
CliXml test.Import-CliXML.test follow-up for WinBlue: 161470 - Export-CliXml errors in WhatIf scenarios
```powershell
Object reference not set to an instance of an object.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/clixml.tests.ps1: line 169
169:             Export-Clixml -Path $testPath -InputObject "string" -WhatIf

```
XmlCommand DRT basic functionality Tests.Import-Clixml -IncludeTotalCount always returns unknown total count
```powershell
Expected strings to be the same, but they were different.
Expected length: 19
Actual length:   0
Strings differ at index 0.
Expected: 'Unknown total count'
But was:  ''
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/XMLCommand.Tests.ps1: line 120
120: 		$out[0].ToString() | Should -BeExactly "Unknown total count"
```
XmlCommand DRT basic functionality Tests.Import-Clixml -First and -Skip work together for simple types
```powershell
Expected 2, but got 4.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/XMLCommand.Tests.ps1: line 126
126: 		$out.Count | Should -Be 2
```
XmlCommand DRT basic functionality Tests.Import-Clixml -First and -Skip work together for collections
```powershell
Expected 2, but got 4.
at <ScriptBlock>, /Users/vsts/agent/2.153.2/work/1/s/test/powershell/Modules/Microsoft.PowerShell.Utility/XMLCommand.Tests.ps1: line 136
136: 		$out2.Count | Should -Be 2
```